plugins/sm-operator/sm-operator/templates/operator.yaml

```diff
@@ -0,0 +1,338 @@
+  - get
+  - update
+  - patch
+- apiGroups:
+  - sagemaker.aws.amazon.com
+  resources:
+  - models
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+  - delete
+- apiGroups:
+  - sagemaker.aws.amazon.com
+  resources:
+  - models/status
+  verbs:
+  - get
+  - update
+  - patch
+- apiGroups:
+  - sagemaker.aws.amazon.com
+  resources:
+  - hyperparametertuningjobs
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+  - delete
+- apiGroups:
+  - sagemaker.aws.amazon.com
+  resources:
+  - hyperparametertuningjobs/status
+  verbs:
+  - get
+  - update
+  - patch
+- apiGroups:
+  - sagemaker.aws.amazon.com
+  resources:
+  - models
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+  - delete
+- apiGroups:
+  - sagemaker.aws.amazon.com
+  resources:
+  - models/status
+  verbs:
+  - get
+  - update
+  - patch
+- apiGroups:
+  - sagemaker.aws.amazon.com
+  resources:
+  - processingjobs
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - sagemaker.aws.amazon.com
+  resources:
+  - processingjobs/status
+  verbs:
+  - get
+  - patch
+  - update
+- apiGroups:
+  - sagemaker.aws.amazon.com
+  resources:
+  - trainingjobs
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+  - delete
+- apiGroups:
+  - sagemaker.aws.amazon.com
+  resources:
+  - trainingjobs/status
+  verbs:
+  - get
+  - update
+  - patch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: sagemaker-k8s-operator-proxy-role
+rules:
+- apiGroups:
+  - authentication.k8s.io
+  resources:
+  - tokenreviews
+  verbs:
+  - create
+- apiGroups:
+  - authorization.k8s.io
+  resources:
+  - subjectaccessreviews
+  verbs:
+  - create
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: sagemaker-k8s-operator-leader-election-rolebinding
+  namespace: {{ .Release.Namespace | default "sagemaker-k8s-operator-system" | quote }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: sagemaker-k8s-operator-leader-election-role
+subjects:
+- kind: ServiceAccount
+  name: {{ .Values.namespace }}
+  namespace: {{ .Values.namespace }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: sagemaker-k8s-operator-manager-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: sagemaker-k8s-operator-manager-role
+subjects:
+- kind: ServiceAccount
+  name: {{ .Values.namespace }}
+  namespace: {{ .Values.namespace }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: sagemaker-k8s-operator-proxy-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: sagemaker-k8s-operator-proxy-role
+subjects:
+- kind: ServiceAccount
+  name: {{ .Values.namespace }}
+  namespace: {{ .Values.namespace }}
+---
+apiVersion: v1
+kind: Service
```

Comment:
same in all these places.. missing helm labels..  I am not going to add comment on every item

Reply:
Show me an example and I can implement